### PR TITLE
Add the #chapel140 programs

### PR DESCRIPTION
**Pull request by bradcray**
Here, I'm adding the programs that we've been tweeting as 140-character (#chapel140) programs for safekeeping.

### CODE REVIEW

**Comment by bradcray** [comment]
@daviditen : Would you like to review this one?  As contributors, @tmacd9 , @mppf , @cassella , and @ben-albrecht may also want to take a peek.

File: test/studies/twitter/sleepsort.chpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+use Time;
```
Comment (daviditen) [inline test/studies/twitter/sleepsort.chpl]: I have a feeling this program might turn into a maintenance headache when some system decides that sleeping for 3 seconds is a fine reaction to a sleep(1) call.  Hopefully not though.

**Comment by daviditen** [comment]
Looks good!

File: test/studies/twitter/sleepsort.chpl (hunk 1 above)
Comment (bradcray) [inline test/studies/twitter/sleepsort.chpl]: Could be.  I'll move to a smaller set of more distantly spaced integers if it does cause problems.  I'm also wondering whether we'll have any portability problems with the file.length logic in the check140*.chpl tests or generally with survey.chpl.

**Pull request merged**